server/routes/account_management.py

```
@@ -176,15 +185,27 @@ def _resolve_row(row):
         # ------------------------------
-        # 4) Kubectl agent connections
+        # 4) GitHub fallback (App installs may not have user_tokens rows)
+        # ------------------------------
+        if "github" not in accounts:
+            try:
+                from routes.connector_status import _check_github
+                result = _check_github(user_id, app_runtime_ready=True)
+                if result.get("connected"):
+                    accounts["github"] = {"isConnected": True, "name": "GitHub", "displayText": "GitHub"}
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)